tests/rkt_api_service_test.go

```diff
@@ -406,3 +423,136 @@ func TestAPIServiceListInspectImages(t *testing.T) {
 		checkImageDetails(t, ctx, m)
 	}
 }
+
+func TestAPIServiceCroup(t *testing.T) {
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	svc := startAPIService(t, ctx)
+	defer stopAPIService(t, svc)
+
+	c, conn := newAPIClientOrFail(t, "localhost:15441")
+	defer conn.Close()
+
+	patches := []string{
+		fmt.Sprintf("--exec=/inspect --serve-http=localhost:%d --serve-http-timeout=%d", testServerPort, testServerTimeout),
```

Comment:
Then, you would not need to listen on port 3727 on the host net namespace (it might be used by someone else).